dockerfiles/tools/download-docker-assets.sh

```diff
@@ -70,66 +84,65 @@ handle_user_input() {
     fi
 }
 
-authenticate_user() {
+setup_authentification() {
```

Comment:
(Mon erreur, désolé!)

```suggestion
setup_authentication() {
```